protocols/imap/src/test/resources/logback-test.xml

```diff
@@ -1,24 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
 
-        <contextListener class="ch.qos.logback.classic.jul.LevelChangePropagator">
-                <resetJUL>true</resetJUL>
-        </contextListener>
-
-        <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
-                <encoder>
-                        <pattern>%d{HH:mm:ss.SSS} %highlight([%-5level]) %logger{15} - %msg%n%rEx</pattern>
-                        <immediateFlush>false</immediateFlush>
-                </encoder>
-        </appender>
-
-        <root level="ERROR">
-                <appender-ref ref="CONSOLE" />
-        </root>
-
-
-        <logger name="org.apache.james" level="DEBUG" >
-                <appender-ref ref="CONSOLE" />
-        </logger>
```

Comment:
don't keep this in debug?

Reply:
No, debug is too much IMO. Logs are more than 100MiB today for a build, it's not manageable anymore. We can very well change the level when needed.